providers/shopify.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "shopify",
+  "defaultService": "default",
+  "services": [
+    {
+      "id": "default",
+      "baseUrl": "https://{SHOP}.myshopify.com"
+    }
+  ],
+  "securitySchemes": [
+    {
+      "id": "apiKey",
+      "type": "apiKey",
+      "in": "header",
+      "name": "X-Shopify-Access-Token"
+    }
+  ],
+  "parameters": [
+    {
+      "name": "SHOP",
+      "description": "Your shop. You can find it in Shopify account settings."
+    },
+    {
+      "name": "api_version",
+      "description": "The version of the API being used.",
+      "default": "2023-04"
```

Comment:
question: I feel like this should be a part of the map, no? As in, the map isn't expected to work with _any_ API version, just the one it was written against, so I don't really understand the point of this being a parameter.